tests/config/SettingConfig.cpp

```diff
@@ -2,8 +2,65 @@
 
 #include "third-party/Catch/single_include/catch2/catch.hpp"
 
+#include "emp/datastructs/vector_utils.hpp"
 #include "emp/config/SettingConfig.hpp"
 
-TEST_CASE("Test SettingConfit", "[config]")
+TEST_CASE("Test SettingConfig", "[config]")
 {
+emp::SettingConfig config;
+
+config.AddSetting<int>("num_runs") = 200;
+config.AddComboSetting<int>("pop_size") = { 100,200,400,800 };
+
+// Test setup
+emp::vector<std::string> setting_names = config.GetSettingMapNames();
+CHECK(emp::Has(setting_names, "num_runs"));
```

Comment:
Why CHECK instead of REQUIRE? I think we're using REQUIRE most everywhere else in the tests. If there's not a particular reason for check probably best to just match

Reply:
We're using a mix. CHECK doesn't stop immediately when it fails (it still runs the rest of the test case and then marks it as failed), so it gives you more information. When we first started using Catch we didn't know about CHECK so we just used REQUIRE everywhere, which is why we've currently still got a higher percentage of REQUIREs than CHECKs, but in reality I think most of them would be better as CHECKs.

Reply:
:heavy_check_mark: maybe at some point in the future (not now) we can go thru and find/replace our REQUIRE('s with CHECK('s